admin/starter/.github/workflows/phpunit.yml

```diff
@@ -11,7 +11,7 @@ jobs:
 
     strategy:
       matrix:
-        php-versions: ['7.4', '8.0']
+        php-versions: ['8.0', '8.1']
```

Comment:
8.2 instead of 8.1?
```suggestion
        php-versions: ['8.0', '8.2']
```

Reply:
Lowest and highest may be better.
Done.